### DISCUSSION TITLE
RPI NetalertX setup within HOST mode, no  networks scanned.

### DISCUSSION THREAD

**Discussion by 2GitHelp** [netalertx/NetAlertX#770] [Q&A]
brand new netalertx install, 
host ip 192.168.3.2 = eth0

setup docker and netalertx per vanilla config on gitub, understood it will try to scan 192.168.1.0/24, so ive changed this

changed to 192.168.3.0/24  and nothing is showing up other than the pihole 192.168.3.2 and the internet gateway 192.168.3.1 

have about 130 devices on this network not showing up, docker is in host mode, and can ping those ip's in which the scans "on schedule" by default, arent showing up, been about 4 days since i walked away from this, what am i doing wrong?

also on occasion after rebuilding the docker container, getting that "loading loop" however trying to work around that. been searching this discussion for similar issues nothing located that matches my issue.

thanks in advance :)

**Comment by jokob-sk** [comment]
Hi there,

Thansk for using NAX!

Make sure you read this carefully, from the sounds of it the subnets setting (the interafce) is not configured correctly: https://github.com/jokob-sk/NetAlertX/blob/main/docs/SUBNETS.md

If you have further questions, please post:

1. your docker run or compose file.
2. the exact value of the scan_subnets setting
